integration/query.test.js

```diff
@@ -70,7 +70,7 @@ it('should return projection', async () => {
     }
   })
 
-  expect(reply).toEqual({
+  expect(reply).toMatchObject({
```

Comment:
Why we are doing this change? `toMatchObject` allows extra fields in the object - is that what we want here?

Reply:
Yes, this has become a common pattern to me, tests should be forward compatible.

I've already been in the situations when I have to update zillion of tests because a new property, unrelated to any of them, has been added to some object.